test/listen.test.ts

```diff
@@ -140,6 +116,37 @@ describe.skipIf(typeof EdgeRuntime === 'string' || typeof document !== 'undefine
       }
     })
 
+    test('sends last-event-id header when reconnecting', async () => {
```

Comment:
buried lede, perhaps, but this change is in a separate commit, so we might want merge with rebase to keep this as a separate item in the commit history.